doc/rbd/iscsi-target-cli-manual-install.rst

```diff
@@ -42,6 +42,10 @@ on each machine that will be a iSCSI gateway:
 -  python flask
 -  pyOpenSSL
 
+1a. Install Common Packages for Debian 10
+==========================
+apt-get update && apt-get install libnl-3-dev libnl-genl-3-dev python2.7-flask python-netifaces
```

Comment:
Technically these are all listed above, correct? Perhaps it would be clearer if there was a yum and apt-get example like is done below -- but include all the required packages and not just a couple.

Reply:
Make your own pull request.